happi-graph.js

```diff
@@ -358,9 +366,22 @@ class HappiGraph extends PolymerElement {
     this.allGroup ? this.allGroup.remove() : (this.debug ? console.log('ALL_GROUP_EMPTY') : 0);
   }
 
+  createMousePositionEventListener(){
+    this.mousePositionEventListener = function (event) {
+      this.mousePosition = {x: event.x, y: event.y};
+    };
+  }
+
+  disconnectedCallback() {
+    this.removeEventListener(this, this.mousePositionEventListener);
```

Comment:
use mousemove instead of "this", `this.removeEventListener('mousemove', this.mousePositionEventListener);`